mobile-app/app/styles.json

```diff
@@ -8670,6 +8670,9 @@
 	"tracking-wide": {
 		"letterSpacing": "0.025em"
 	},
+	"tracking-wide-v2": {
```

Comment:
We don't modifiy this file directly, we 
- add/extend custom tw classes in `tailwind.config.js` 
- cd `mobile-app/app/tailwind.ts` 
- then run `npx create-tailwind-rn`
- this will re-generate a new `styles.json` based on our custom config

Reply:
updated